#
#
# completes reinitialization of communications sec layer 
# used in case of server redirection: without this the server
# (2008) to which session was redirected presents just a black or
# 'welcome' screen and does not talk to us ...
#
# 9 Oct 2012 Jaroslaw.Polok@cern.ch
#
#
diff -Naur rdesktop-1.7.1.org/secure.c rdesktop-1.7.1.lb/secure.c
--- rdesktop-1.7.1.org/secure.c	2011-11-21 11:51:08.000000000 +0100
+++ rdesktop-1.7.1.lb/secure.c	2012-10-08 17:07:01.368290172 +0200
@@ -895,5 +895,6 @@
 	g_server_rdp_version = 0;
 	g_sec_encrypt_use_count = 0;
 	g_sec_decrypt_use_count = 0;
+	g_licence_issued = False;
 	mcs_reset_state();
 }
